packages/nyc-config-babel/README.md

```diff
@@ -14,7 +14,7 @@ Then write a `.babelrc` that looks something like this:
 
 ```json
 {
-  "presets": ["env", "..., etc."],
+  "presets": ["@babel/preset-env", "..., etc."],
```

Comment:
NITPICK: I think `@babel/env` is a valid and shorter alias.

Reply:
Appears so I'm not sure why I previously remember having trouble with this.

Updated for this and your other finding, thanks for the review!